secedgar/client/network_client.py

```
@@ -68,6 +69,17 @@ def batch_size(self, value):
             raise ValueError("Batch size must be positive integer.")
         self._batch_size = value
 
+    @property
+    def rate_limit(self):
+        return self._rate_limit
+
+    @rate_limit.setter
+    def rate_limit(self, value):
+        if value < 0:
+            self._rate_limit = 0
```

we should throw an error though

---

Yeah, you're right